src/components/Modal/DescriptionModal.jsx

```diff
@@ -0,0 +1,43 @@
+import { useNavigate } from "react-router-dom";
+import PropTypes from "prop-types";
+
+import Modal from "../shared/Modal/Modal";
+
+import CONTENTS from "../../utils/ModalContents";
+
+function DescriptionModal({ navigateNextPage, statement }) {
+  const navigate = useNavigate();
+
+  const title = CONTENTS.title[statement];
+  const nextButtonText = CONTENTS.button[statement];
+  const message = CONTENTS.message[statement];
+
+  const handleCloseButtonClick = () => {
+    if (statement === "warning") {
+      navigate("/");
+    } else {
+      navigate("/modal/warning");
+    }
```

Comment:
```suggestion
    if (statement === "warning") {
      navigate("/modal/warning");
    } else {
      navigate("/");
    }
```
실제 기능상 맞는 동작이어도 
`statement`가 `warning` 일 때 기본 주소로 네비게이트 하고 `warning`이 아닐때 `warning` 으로 네비게이트 하는 것은 조금 어색해 보이는 것 같습니다!